api/src/main/java/org/apache/iceberg/io/ParallelIterable.java

```diff
@@ -72,8 +87,8 @@ private ParallelIterator(Iterable<? extends Iterable<T>> iterables,
             }
           }).iterator();
       this.workerPool = workerPool;
-      // submit 2 tasks per worker at a time
-      this.taskFutures = new Future[2 * ThreadPools.WORKER_THREAD_POOL_SIZE];
+      // submit 2 tasks per worker at a time.
```

Comment:
Can you remove the non-functional change on this line? We don't want unnecessary changes to cause commit conflicts.